data/foss2cyber.json

```
@@ -0,0 +1,63 @@
+{
+    "name": "Ishan Velle",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Front-end Web Developer | Self-taught | Designer | Freelancer | Blogger",
+    "avatar": "https://github.com/foss2cyber.png"
```

Please add the missing comma `,` here. 

```suggestion
    "avatar": "https://github.com/foss2cyber.png",
```